interfaces/Config/templates/config_switches.tmpl

```diff
@@ -52,6 +52,11 @@
                         </select>
                         <span class="desc">$T('explain-load_balancing')</span>
                     </div>
+                    <div class="field-pair <!--#if int($have_ssl) == 0 then "disabled" else ""#-->">
```

Comment:
When the user fills in an invalid cipher, SSL connections will no longer work. Is it possible to ignore invalid cipher names?